### ISSUE TITLE
PFS listens to Blockchain Events

### ISSUE THREAD

**Issue by palango** [raiden-network/raiden-pathfinding-service#81]
## Description

**As Josh I want the PFS to**

listen to Blockchain Events

**So that it can**

recreate the network topology and find out about channels deposits.

## Acceptance Criteria

- [ ] `TokenNetworkCreated` events are observed and create new networks inside the PFS
- [ ] `ChannelOpen` and `ChannelClose` events are observes and open/close channels in the respective network
- [ ] `ChannelNewDeposit` events are observed and increase the capacity for the respective channel

## Related story / issue
- 

## Reviewer
-

**Comment by Dominik1999** [comment]
Topology is now saved as a .png with every basic info request. It can be shown to the user

**Comment by Dominik1999** [comment]
Hmm all acceptance criteria are ticked but some tasks are open - that seems inconsistent somehow

**Comment by palango** [comment]
Yes, the user visible stuff is done, but some tests need to be fixed.

**Comment by Dominik1999** [comment]
Maybe we should include - updated tests - to the acceptance criteria?
